src/api/v1/serializers/pizzas-serializer.js

```diff
@@ -53,25 +53,32 @@ const transformRawPizza = (rawPizza) => {
  * @param {string} relationName
  * @param {object} relationKeys
  * @param {string} idField
+ * @param {boolean} [ignoreRelationshipData=false] used when the `data` member of
+ * the relationship should be omitted
  * @returns {object} `options` with `relationName` added
  */
-const addCompoundRelationship = (options, relationName, relationKeys) => {
+const addCompoundRelationship = (
+  options,
+  relationName,
+  relationKeys,
+  ignoreRelationshipData = false,
+) => {
   const idField = 'id';
   const resourceUrl = resourcePathLink(apiBaseUrl, relationName);
   options.attributes = _.concat(options.attributes, relationName);
   options[relationName] = {
-    ref: (collection, field) => field[idField],
+    ref: (collection, field) => (field && idField in field ? field[idField] : undefined),
```

Comment:
I guess `idField in field` part is not necessary?

Reply:
yep, looks like it. Good catch.